plugins/Goals/API.php

```diff
@@ -230,8 +230,6 @@ protected function getItems($recordName, $idSite, $period, $date, $abandonedCart
         $archive   = Archive::build($idSite, $period, $date, $segment);
         $dataTable = $archive->getDataTable($recordNameFinal);
 
-        $dataTable->filter('Sort', array(Metrics::INDEX_ECOMMERCE_ITEM_REVENUE));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice